startup/92-optimization.py

```diff
@@ -260,27 +256,34 @@ def calc_velocity(motors, dists, velocity_limits, max_velocity=None):
         upper_velocity_bounds = []
         for j in range(len(velocity_limits)):
             upper_velocity_bounds.append(velocity_limits[j][1])
-        # find max distance to move
-        max_dist = np.max(dists)
-        max_dist_index = dists.index(max_dist)
-        max_dist_vel = upper_velocity_bounds[max_dist_index]
+        # find max distances to move and pick the slowest motor of those with max dists
+        max_dist_lowest_vel = np.where(dists == np.max(dists))[0]
+        if len(max_dist_lowest_vel) == 1:
+            motor_index_to_use = max_dist_lowest_vel[0]
+        else:
+            max_dist_to_move = -1
+            for j in max_dist_lowest_vel:
+                if dists[j] >= max_dist_to_move:
+                    max_dist_to_move = dists[j]
+                    motor_index_to_use = j
```

Comment:
I think the first case of one element in `max_dist_lowest_vel` is covered by the else condition, so it feels the `if...else` can be skipped by leaving just the code in the `else` block.